docs/ise_compatibility.md

```diff
@@ -0,0 +1,118 @@
+# ISE Compatibility
+
+While the PowerShell extension for VSCode does not seek
+100% compatibility/reproduction of features in the PowerShell ISE,
+there are features in place to make the VSCode experience more natural
+for users of the ISE.
+
+This document tries to list settings you can configure in VSCode
+to make the user experience a bit more familiar compared to the ISE.
+
+## Key bindings
+
+| Function                              | ISE Binding                  | VSCode Binding                              |
+| ----------------                      | -----------                  | --------------                              |
+| Interrupt and break debugger          | <kbd>Ctrl</kbd>+<kbd>B</kbd> | <kbd>F6</kbd>                               |
+| Execute current line/highlighted text | <kbd>F8</kbd>                | <kbd>F8</kbd>                               |
+| List available snippets               |                              | <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>J</kbd> |
+
+## Tab-completion
```

Comment:
```suggestion
## Tab completion
```

Looks off to me, though if the hyphenation is common with that then just ignore me.